test/views/sections/items/ItemPackagingSealChoiceViewSpec.scala

```diff
@@ -44,15 +44,16 @@ class ItemPackagingSealChoiceViewSpec extends SpecBase with ViewBehaviours {
         val view = app.injector.instanceOf[ItemPackagingSealChoiceView]
         val form = app.injector.instanceOf[ItemPackagingSealChoiceFormProvider].apply()
 
-        implicit val doc: Document = Jsoup.parse(view(form, testOnwardRoute, "Aerosol").toString())
+        implicit val doc: Document = Jsoup.parse(view(form, testOnwardRoute, "Aerosol", 0, 0, "1", false).toString())
```

Comment:
Could you name these parameters? `0`, `0`, "1" is a bit hard to read without knowing what the parameters are for